src/qml/MapCanvas.qml

```diff
@@ -60,106 +67,136 @@ Item {
     freeze: false
   }
 
-  PinchArea {
-    id: pinchArea
+    TapHandler {
+      grabPermissions: PointerHandler.ApprovesTakeOverByAnything
 
-    anchors.fill: parent
+      property bool longPressActive: false
 
-    onPinchStarted: {
-      freeze('pinch')
-    }
+      property var timer: Timer {
+          property var firstClickPoint
+          interval: mouseDoubleClickInterval
+          repeat: false
 
-    onPinchUpdated: {
-      mapCanvasWrapper.zoom( pinch.center, pinch.previousScale / pinch.scale )
-      mapCanvasWrapper.pan( pinch.center, pinch.previousCenter )
-      mapArea.panned()
-    }
+          onTriggered: {
+              clicked(firstClickPoint)
+          }
+      }
+
+      onTapCountChanged: {
```

Comment:
@m-kuhn why not using https://doc.qt.io/qt-5/qml-qtquick-taphandler.html#doubleTapped-signal ?

Reply:
Always before a doubleTapped is fired, a single tap is fired as well